teams/dac-migration-dss.yaml

```diff
@@ -0,0 +1,8 @@
+DAC-Migration-DSS:
```

Comment:
If you want to prettify this, you could remove the hyphens. The slug would still remain `DAC-Migration-DSS`, but the display would be a bit nicer.